pyproject.toml

```
@@ -22,7 +22,7 @@ classifiers = [
 ]
 dependencies = [
     "Sphinx>=4.2.0",
-    "pydata-sphinx-theme>=0.14,<0.15",
+    "pydata-sphinx-theme>=0.15",
```

```suggestion
    "pydata-sphinx-theme>=0.15.4<0.16",
```
I would put this one better =) since it is the version that fixed the issue - and block the next minor version